reference/v8.json

```diff
@@ -43,6 +43,38 @@
       "doc": "Default pitch, in degrees. Zero is perpendicular to the surface, for a look straight down at the map, while a greater value like 60 looks ahead towards the horizon. The style pitch will be used only if the map has not been positioned by other means (e.g. map options or user interaction).",
       "example": 50
     },
+    "light": {
+      "anchor": {
+        "type": "enum",
+        "default": "viewport",
+        "values": [
+          "map",
+          "viewport"
+        ],
+        "doc": "Whether extruded geometries are lit relative to the map or viewport. The `light.anchor` property will only be used if it has not been set by map options upon initialization.",
+        "example": "map"
+      },
+      "direction": {
```

Comment:
Given the current documentation, I think you should strongly consider renaming the property to "position", even if the _goal_ of setting this property is to change the lighting direction. (You can say as much in the documentation.)